### DISCUSSION TITLE
More functions for Insert > Layout & another useful layout feature

### DISCUSSION THREAD

**Discussion by I2rys** [jgraph/drawio-desktop#884] [Ideas]
* [X] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [X] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
Not really but these features are really useful.

**Describe the solution you'd like**
So when you go to Insert > Layout and picked your layout. A feature where you can delete, use the specific boxes that you want like the normal functions. Second feature which is for the context where If you select multiple boxes and right click there is a option there where you can go to the layout mode thingy where It automatically layout your boxes and can add more boxes then save/cancel.

**Describe alternatives you've considered**
No alternatives.

**Additional context**
No additional context.

**Comment by davidjgraph** [comment]
This is too general for a feature request, I'll move to a discussion.
